tests/cms/views/organizations/test_organization_form.py

```diff
@@ -0,0 +1,140 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from django.test.client import Client
+    from pytest_django.fixtures import SettingsWrapper
+
+import pytest
+from django.test.client import Client
+from django.urls import reverse
+
+from integreat_cms.cms.models import MediaFile, Organization, Page, POI, Region
+from tests.conftest import ANONYMOUS, MANAGEMENT, STAFF_ROLES
+
+
+@pytest.mark.django_db
+def test_poi_form_shows_no_contents(
+    load_test_data: None,
+    login_role_user: tuple[Client, str],
+    settings: SettingsWrapper,
+) -> None:
+    """
+    Test that no contents are shown in the organization form if it does not have any content assinged
+    """
+    client, role = login_role_user
+
+    # Test for english messages
```

Comment:
Could you please change this comment to describe why we are doing this? :)

Reply:
Hmm, I can't imagine what you exactly expect but this part is updated too.

Reply:
Yes, makes it more clearer, thank you.
I have been wondering why we didn't pick German :)

Reply:
I started using English for testing messages just because there were already tests doing so. We could of course use German but not all of potential external contributers of our open repository speak German and tests can fail because something is not working in the translation but not the function itsself.